### PR TITLE
Apply new vocabulary

### PR DESCRIPTION
**Pull request by clementbiron**
That changetset apply the vocabulary, replacing `instance`by `collection` and `type` to `terms type`.

### CODE REVIEW

**Comment by MattiSG** [comment]
For the next review, I would prefer that @Ndpnt takes over, as he is more familiar with the technical implementation of the vocabulary 🙂

File: src/modules/Common/components/ServicesList.tsx [hunk 1]
```diff
@@ -25,21 +25,21 @@ const ServicesList: React.FC<ServicesListProps> = ({
       <Column width={100} title={title} subtitle={subtitle}>
         <ul className={s.listServices_items}>
           {Object.keys(services || {}).map((serviceName: string) => {
-            const documentTypes = (services || {})[serviceName];
+            const documentTermsTypes = (services || {})[serviceName];
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply src/modules/Common/components/ServicesList.tsx:28]: It is a way for me to remind the context but it is not necessary.

File: src/pages/subscribe.tsx [hunk 2]
```diff
@@ -25,7 +25,7 @@ export default function SubscribePage({ services }: any) {
       await api.post(`/api/subscribe`, {
         email: data.email,
         service: data.service,
-        documentType: data.documentType,
+        documentTermType: data.documentTermType,
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply src/pages/subscribe.tsx:28]: Thank you for pointing this out to me, your comment and a conversation with @Ndpnt  helped me to understand how to name terms types in singular.

File: src/modules/Common/components/SubscribeForm.tsx [hunk 3]
```diff
@@ -38,32 +38,32 @@ const SubscribeForm = ({
...
-    if (previousService !== service && !!previousService && !!documentType) {
-      reset({ documentType: undefined, service, email, consent });
+    if (previousService !== service && !!previousService && !!termsType) {
+      reset({ termsType: undefined, service, email, consent });
     }
-  }, [previousService, documentType, service, reset, email, consent]);
+  }, [previousService, termsType, service, reset, email, consent]);
 
-  // reset document type and service when submit as occured
+  // reset terms type and service when submit as occured
```
Comment (MattiSG) [inline src/modules/Common/components/SubscribeForm.tsx:59]:
```suggestion
  // reset terms type and service when submit has occurred
```

File: src/modules/Github/api/index.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-const DOCUMENT_TYPES_URL = 'https://opentermsarchive.org/data/api/list_documentTypes/v1/';
+const TERMS_TYPES_URL = 'https://opentermsarchive.org/data/api/list_termsTypes/v1/';
```
[the remark this answers is not in the document]
Comment (clementbiron) [reply src/modules/Github/api/index.ts]: Wooo I was totally focused on the name of the variable and I didn't see that my changes concerned its value, thanks for your attention!

File: locales/fr/stats.json [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {
   "numbers.block1.desc": "répartis sur plusieurs industries, langues et juridictions",
   "numbers.block1.title": "400+ services",
-  "numbers.block2.desc": "classés selon une taxonomie d'une cinquantaine de types",
+  "numbers.block2.desc": "classés selon une taxonomie d'une cinquantaine de types de termes",
```
Comment (Ndpnt) [inline locales/fr/stats.json:4]: I'm not sure that we should translate "terms" to "termes" in French? @MattiSG

**Review by Ndpnt — APPROVED** [review]

**Pull request merged**